framework/src/controller/controller.js

```diff
@@ -228,12 +232,17 @@ class Controller {
 
 		const child = child_process.fork(program, parameters);
 
-		child.on('exit', (code, signal) => {
+		this.childrenList.push(child);
```

Comment:
Where do we handle killing all child processes when a SIGINT received?

Reply:
`Controller.cleanup`

Reply:
Great, just one more comment. I think we should also pass the kill signal to child processes. 

https://github.com/LiskHQ/lisk/blob/ec96e8de10104a360b9f0e06bee9a7f55f8806f2/framework/src/controller/controller.js#L281

Reply:
By default it sends `SIGTERM` signal. What you reckon?

Reply:
I think if main application was killed with SIGINT, children should be exited with SIGINT too.

Reply:
To do that we need to change the way we're exiting the application. We can improve that in the next version. Could you open an issue for that, pls?

Reply:
https://github.com/LiskHQ/lisk/issues/3185 @lsilvs @MaciejBaj